test/data/manifests.js

```diff
@@ -0,0 +1,1036 @@
+window.manifests = {
```

Comment:
Should this be committed? Previously this was generated as part of the build process.